native/media-cache-wrapper/Cargo.toml

```
@@ -10,16 +10,18 @@ rust-version = "1.70"
 [dependencies]
 async-trait = "0.1.60"
 base64ct = { version = "1.6.0", features = ["alloc"] }
-rmp-serde = "1.1.2"
 matrix-sdk-base = { workspace = true }
 matrix-sdk-store-encryption = { workspace = true }
+rmp-serde = "1.1.2"
+ruma-common = { workspace = true }
+ruma-events = { workspace = true }
 serde_json = "1"
 tracing = "0.1.40"
 
 [dev-dependencies]
+anyhow = "1.0.79"
 matrix-sdk-sqlite = { workspace = true }
 matrix-sdk-test = { workspace = true }
-tokio = "1.35.1"
 tempfile = "3.9.0"
-anyhow = "1.0.79"
+tokio = "1"
```

we need to keep that at a minimum as well
```suggestion
tokio = "1.35.1"
```

---

Fixed by https://github.com/acterglobal/a3/pull/1346/commits/cfb67aaf65bfe250bf39a8016ca53eb22c8f3e51